renderdoc/driver/vulkan/wrappers/vk_get_funcs.cpp

```diff
@@ -1364,6 +1379,7 @@ void WrappedVulkan::vkGetImageSubresourceLayout2KHR(VkDevice device, VkImage ima
 {
   ObjDisp(device)->GetImageSubresourceLayout2KHR(Unwrap(device), Unwrap(image), pSubresource,
                                                  pLayout);
+  AdjustSubresourceHostMemcpySizeQuery(pSubresource, pLayout);
```

Comment:
I think this can now be simplified to:

```
  VkSubresourceHostMemcpySize *memcpySize =
      (VkSubresourceHostMemcpySize*)FindNextStruct(
          pLayout, VK_STRUCTURE_TYPE_SUBRESOURCE_HOST_MEMCPY_SIZE);
  if(memcpySize)
    memcpySize->size = 64;
```

Rather than having a separate helper that does its own pNext chasing for a simple assignment.